features/svg-animations.yml

```
@@ -0,0 +1,97 @@
+name: SVG animations
+description: The `<animate>`, `<animateMotion>`, and `<animateTransform>` SVG elements declaratively animate SVG elements. Also known as SMIL.
```

```suggestion
description: The `<animate>`, `<animateMotion>`, and `<animateTransform>` SVG elements animate an SVG element's attributes over time. Also known as SMIL.
```

Here I also think we should mention the over time concept, and maybe also mention that it's the attributes that get animated (if I'm not mistaken)

---

Can you explain the significance of differentiating between the element and the element's attributes? Perhaps a poor analogy, but I think of it as "You press the gas pedal to turn the wheels" vs "You press the gas pedal to move the car"- the intended outcome is moving the car, and the wheels are somewhat incidental.

---

I think what prompted me to talk about attributes is that we describe CSS animations as animating an element's _styles_.
So, I felt compelled to mention exactly _what_ were SVG animations animating.

Animating an element feels quite abstract to me, but perhaps it's just me. I won't block on this. If you prefer "animate SVG elements", go for it.

---

I'm going to weigh in here and say that I prefer _elements_ over _attributes_, since it covers the whole feature. This is because `<animateMotion>` is the odd one out, in that it doesn't obviously animate an element's attribute.